website/src/css/global.css

```diff
@@ -156,6 +157,43 @@
   margin-right: 8px;
 }
 
+@media screen and (max-width: 996px) {
+  .navbarSearchContainer_Bca1 {
+    padding: 4px 8px !important;
+    margin-right: 4px;
+    border-right: none !important;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .DocSearch-Button {
+    width: 180px !important;
+  }
+  
+  .navbar {
+    padding: 0px 12px;
+  }
+}
+
+@media screen and (max-width: 150px) {
+  .DocSearch-Button {
+    width: 120px !important;
```

Comment:
I think there was some misunderstaing. We needed to make this from 120px to 150px. Can you please undo the last change?